src/OMSimulatorLib/OMSimulator.h

```diff
@@ -73,15 +73,15 @@ oms_status_enu_t oms3_getConnections(const char* cref, oms3_connection_t*** conn
 oms_status_enu_t oms3_addConnection(const char* crefA, const char* crefB);
 oms_status_enu_t oms3_addBus(const char* cref);
 oms_status_enu_t oms3_addConnectorToBus(const char* busCref, const char* connectorCref);
+oms_status_enu_t oms3_addTLMBus(const char* cref, const char* domain, const int dimensions, const oms_tlm_interpolation_t interpolation);
+oms_status_enu_t oms3_addConnectorToTLMBus(const char* busCref, const char* connectorCref, const char *type);
```

Comment:
What is wrong with `oms_tlm_connector_type_enu_t`?

Reply:
That enum would need to contain almost 200 entries, to cover all combinations of dimensions, causalities and interpolation methods. This feels unrealistic. I added a list of strings with accepted variable types for each combination instead.